tests/BaGet.Tests/BaGet.Tests.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="NuGet.Protocol" Version="5.0.0-rtm.5856" />
+    <PackageReference Include="NuGet.Protocol" Version="5.3.1" />
```

Comment:
This now uses a variable `$(NuGetPackageVersion)`. That is defined here:

https://github.com/loic-sharma/BaGet/blob/05bfc61e1ff86517d2e4571f12524275c2355a0d/tests/Directory.Build.props#L18